b/id.go

```diff
@@ -0,0 +1,85 @@
+package xidb
```

Comment:
Can you add some package level do to explain what this package is about and provide some examples on how to use it?

Reply:
ofc! I just wanted to wait and see if i was on the right track